README.md

```diff
@@ -1,70 +1,38 @@
-### Docker users, see [Moby and Docker](https://mobyproject.org/#moby-and-docker) to clarify the relationship between the projects
```

Comment:
I think this sentence is still good to keep

Reply:
It is still correct, but I really wanted to not begin with something about Docker, I wanted the project to stand on its own. I think for anyone reading this it should be clear enough.